### DISCUSSION TITLE
Error using imcopy

### DISCUSSION THREAD

**Discussion by suvadip31** [iraf-community/iraf#143] [Q&A] [answered]
Hi all, 

I am trying to copy a certain portion of a fits image into a new output image. I am using:
`imcopy input.fits[1:1024, 1:1024] output.fits` to copy only the specified section of the input image. But it is showing me an error: `ERROR: FXF: number is not a decimal`

Any help is highly appreciated. 

I am using IRAF v2.16.1+2018.11.01 on ubuntu 18.04

**Comment by olebole** [comment]
I just tested it, and I got no error, using an extensionless fits file.
Maybe your file is a MEF file and you need to specify the extension?

    ecl> imcopy input[0][1:1024, 1:1024] output

**Comment by suvadip31** [comment]
Do I need to put `[0]` before specifying the section of the image?

**Comment by olebole** [comment, marked as answer]
If it is a MEF (Multi Extension Fits) file and the interesting part in in the primary extension: yes. If it is in the first extension, put a `[1]` there.

**Comment by suvadip31** [comment]
Thank you so much! I just tested it and it gives me the correct output. I have used `[0]`.
